.lint.config.json

```
@@ -83,7 +86,10 @@
 				"yarn run refactor"
 			],
 			"allowlist": [
-				"frontend/www/.*\\.ts$"
+				"frontend/www/.*\\.ts$",
+				"cypress/e2e/.*\\.ts$",
+				"cypress/support/.*\\.ts$",
+				"cypress/support/pageObjects/.*\\.ts$"
```

This pattern should be enough to apply the linter to all `.ts` files contained in the cypress directory:
```suggestion
				"cypress/.*\\.ts$"
```